Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -592,6 +604,31 @@ void MainWidget::initialize_menubar(GUI::Window& window)
             }
         }));
 
+    m_layer_via_copy = GUI::Action::create(
+        "Layer via Copy", { Mod_Ctrl | Mod_Shift, Key_C }, g_icon_bag.new_layer, [&](auto&) {
+            auto add_layer_success = current_image_editor()->add_new_layer_from_selection();
+            if (!add_layer_success) {
+                GUI::MessageBox::show_error(&window, String::formatted("Unable to create layer from selection"));
```

Comment:
See previous comment; if we had an actual error from the image editor call, we could display its text here and give the user more information.

Reply:
I've updated this to check for is_error(). Thanks for the feedback and having me give this a second look.